recipes/pyxid/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyxid" %}
+{% set version = "1.0" %}
+{% set sha256 = "e689bce2ed8ed4bff7174a3a79f8b639ebf6356807aec7e9da481217313214ff" %}
+
+package:
+  name: pyxid
+  version: {{ version }}
+
+source:
+  fn: pyxid-{{ version }}.tar.gz
+  url: https://github.com/cedrus-opensource/pyxid/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyserial >=2.5
```

Comment:
Sorry for the delay! Yes, `pyserial` is needed at build time (dependency failure without it), so we need to leave it in.

```
Installed /Users/kastman/miniconda/conda-bld/pyxid_1488744704259/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python2.7/site-packages/pyxid-1.0-py2.7.egg
Processing dependencies for pyxid==1.0
Searching for pyserial>=2.5
Traceback (most recent call last):
  File "setup.py", line 26, in <module>
    "Operating System :: POSIX",
  File "/Users/kastman/miniconda/conda-bld/pyxid_1488744704259/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python2.7/distutils/core.py", line 151, in setup
...
...
```